server/controllers/UsersController.js

```diff
@@ -92,12 +93,57 @@ class UsersController {
           });
         }
       })
-      .catch(error => res.status(500).json({
+      .catch(err => res.status(500).json({
         errors: {
-          message: ['error reading user table', `${error}`]
+          message: [err.message]
         }
       }));
   }
+
+  /**
+    * @description - This method logs in user and return a token.
+    * @param {object} req - The request object bearing the email and password.
+    * @param {object} res - The response object that is returned as json.
+    * @returns {object} - The json object with message.
+    * @memberOf UserController
+    * @static
+    */
+  static verifyUser(req, res) {
+    const { token } = req.query;
+    const { id } = verifyToken(token);
+    User
+      .findByPk(id)
+      .then((user) => {
+        if (user.confirmEmail !== false) {
```

Comment:
I believe doing this 
```
if (confirmEmail) {
   console.log('this is true)
}
``` 

is same as what you have done unless you are trying to check for a specific value. Can you provide more clarity on your approach?